tests/Unit/Service/Phishing/DateCheckTest.php

```diff
@@ -65,4 +65,24 @@ public function testInTheFuture(): void {
 		$this->assertTrue($result->isPhishing());
 	}
 
+	public function testInvalidDate(): void {
+
+		$this->time->expects($this->exactly(2))
+			->method('getDateTime')
+			->withConsecutive(
+				['now'],
+				['invalid date']
+			)
+			->will(
+				$this->onConsecutiveCalls(
+					$this->returnValue(new \DateTime('now')),
+					$this->throwException(new \DateException())
+				)
+			);
```

Comment:
PHPUnit is removing withConsecutive

See https://stackoverflow.com/questions/75389000/replace-phpunit-method-withconsecutive-abandoned-in-phpunit-10 for a similar alternative approach